src/modules/Dropdown/Dropdown.js

```diff
@@ -335,9 +335,13 @@ export default class Dropdown extends Component {
   componentWillMount() {
     if (super.componentWillMount) super.componentWillMount()
     debug('componentWillMount()')
-    const { open, value } = this.state
+    const { value } = this.state
 
     this.setValue(value)
+  }
+
+  componentDidMount() {
+    const { open } = this.state
```

Comment:
Because open() sets state, this results in setting state on mount which is an antipattern.  We have a lint rule for this enabled, although, because it is indirect and abstracted away, it is not picking up on it.

I think we should instead add guarding logic where the focus() is called and only call it if there is a node present.